src/projections/projectResearcherVouchedPackages.js

```diff
@@ -0,0 +1,52 @@
+import { ethers } from 'ethers'
+import { normalizeAddr } from '~/utils/normalizeAddr'
+
+export function projectResearcherVouchedPackages (address, events) {
+  let currentVouchTotal
+
+  address = normalizeAddr(address)
+
+  const result = {
+    packages: {}
+  }
+  
+
+  for (let i in events) {
+    // use extraTopics to filter events based on user's eth address!
```

Comment:
I'm not sure how to do this. I attempted creating a new query with $address as the variable and passing it in to filter the events, but it didn't seem to work. 

I then tried without a var and just hard-coded both of these addresses into the gql:

```
0x8f7f92e0660dd92eca1fad5f285c4dca556e433e
0x8f7F92e0660DD92ecA1faD5F285C4Dca556E433e
```

With no luck in the Apollo Chrome inspector. Will check in with you tomorrow on it! Might be something to do with Apollo Link Ethereum

Reply:
Yeah let's review tomorrow.  I can swing by yours.